ec2utils/ec2utilsbase/setup.py

```diff
@@ -46,11 +46,11 @@
             '': 'lib',
         },
         namespace_packages = ['ec2utils'],
-        classifiers=(
+        classifiers=[
             'Development Status :: 5 - Production/Stable',
             'Intended Audience :: Developers',
             'Natural Language :: English',
-            'License :: OSI Approved :: GPL-3.0+',
+            'License :: OSI Approved :: GNU General Public License v3 (GPLv3)',
```

Comment:
License classifier should be:

License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)